src/components/app-rating-mobile/AppRatingMobile.tsx

```diff
@@ -11,16 +11,17 @@ import { TotalReviews, UserRoleEnum } from '~/types'
 
 interface AppRatingMobileProps extends RatingProps {
   reviewsCount: TotalReviews
+  showStar?: boolean
 }
 
-const AppRatingMobile: FC<AppRatingMobileProps> = ({ value, reviewsCount }) => {
+const AppRatingMobile: FC<AppRatingMobileProps> = ({ value, reviewsCount, showStar = true }) => {
```

Comment:
I think show star by default should be false, and when you pass this prop it will be true, or rename it to hideStar